src/components/header/index.tsx

```diff
@@ -1,11 +1,43 @@
-import React from "react";
+import { Alignment, Button, Icon, Navbar as BPNavbar } from "@blueprintjs/core";
+import * as React from "react";
+import { Link } from "react-router-dom";
+import HeaderPatientName from "components/header/headerPatientName";
+
 import "./style.css";
 
-const Header = () => {
+interface Props {
+  patientId: string;
+}
+
+const Header = ({ patientId }: Props) => {
   return (
-    <div className="App-header">
-      <h2 className="App-title">Welcome to PatientVisualizer !</h2>
-    </div>
+    <BPNavbar id="navbar" className="bp3-dark">
+      <BPNavbar.Group align={Alignment.LEFT}>
+        <BPNavbar.Heading>
+          <Link
+            style={{ textDecoration: "none", color: "white", display: "flex" }}
+            to="/"
+          >
+            <img
+              id="logoNavbar"
+              src={process.env.PUBLIC_URL + "/arkhn_logo_only_white.svg"}
```

Comment:
you don't need the public url here, you can just have src="arkhn_logo_only_white.svg" since it's at the root of the "public" folder